packages/shared/lib/services/connection.service.ts

```diff
@@ -502,25 +502,16 @@ class ConnectionService {
 
     public async listConnections(
         environment_id: number,
-        limit: number | null = 20,
-        offset = 0,
-        integration?: string
+        connectionId?: string
```

Comment:
I think the problem was here. @hassan254-prog can you revisit the PR, making sure the behavior of listConnections is not altered. Thank you very much.